types/react-bootstrap/components/OverlayTrigger.d.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Why are you adding this files here, instead of using the `@types/react-bootstrap`

Reply:
Nevermind, I saw the comments.